mrack.spec

```diff
@@ -1,59 +1,145 @@
 # Created by pyp2rpm-3.3.5
-%global srcname mrack
-
-Name:           %{srcname}
+Name:           mrack
 Version:        1.8.1
 Release:        1%{?dist}
 Summary:        Multicloud use-case based multihost async provisioner for CIs and testing during development
 
 License:        Apache License 2.0
 URL:            https://github.com/neoave/mrack
-Source0:        https://github.com/neoave/mrack/releases/download/v%{version}/mrack-%{version}.tar.gz
+Source0:        https://github.com/neoave/mrack/releases/download/v%{version}/%{name}-%{version}.tar.gz
 BuildArch:      noarch
 
 BuildRequires:  python3-devel
-BuildRequires:  python3-AsyncOpenStackClient
-BuildRequires:  beaker-client
-BuildRequires:  python3-boto3
-BuildRequires:  python3-botocore
 BuildRequires:  python3-click
 BuildRequires:  python3-pyyaml
 BuildRequires:  python3-setuptools
 
+# coma separated list of provider plugins
+%global provider_plugins aws,beaker,openstack,podman,virt
+
+Requires:       %{name}-cli == %{version}-%{release}
+Requires:       python3-%{name}lib == %{version}-%{release}
+Requires:       python3-%{name}-aws == %{version}-%{release}
+Requires:       python3-%{name}-beaker == %{version}-%{release}
+Requires:       python3-%{name}-openstack == %{version}-%{release}
+Requires:       python3-%{name}-podman == %{version}-%{release}
+Requires:       python3-%{name}-virt == %{version}-%{release}
+
+%global __requires_exclude asyncopenstackclient
```

Comment:
Please document why you are doing this :-)

Reply:
Updated